src/service/performance-impl.js

```diff
@@ -81,7 +81,7 @@ export class Performance {
     this.viewer_ = null;
 
     /** @private {?./resources-impl.Resources} */
-    this.resources = null;
+    this.resources_ = null;
```

Comment:
we reassign with an underscore in `coreServicesAvailable` which is where things start executing so declaration was just bad but thankfully didn't affect functionality.